aucpace/Cargo.toml

```diff
@@ -0,0 +1,56 @@
+[package]
+name = "aucpace"
+version = "0.1.0"
+authors = ["Sam Leonard <tritoke@protonmail.com>"]
+description = "AuCPace protocol implementation"
+documentation = "https://docs.rs/aucpace"
+homepage = "https://github.com/RustCrypto/PAKEs"
+repository = "https://github.com/RustCrypto/PAKEs/tree/master/aucpace"
+license = "MIT OR Apache-2.0"
+keywords = ["crypto", "pake", "authentication"]
+categories = ["cryptography", "authentication"]
+exclude = [".gitignore"]
+readme = "README.md"
+edition = "2021"
+rust-version = "1.60"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+curve25519-dalek = { version = "4.0.0-rc.1", default-features = false, features = ["digest", "rand_core"] }
+password-hash = { version = "0.4", default-features = false, features = ["rand_core"] }
```

Comment:
This is out-of-date
```suggestion
password-hash = { version = "0.5", default-features = false, features = ["rand_core"] }
```

Reply:
Thanks I'll add a commit to bump these two dependencies.